### PR TITLE
Internal: Documentation of `document/dynamic` component. [ED-5574]

### PR DESCRIPTION
**Pull request by iNewLegend**

### CODE REVIEW

File: docs/assets/dev/js/editor/document/dynamic/readme.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
+## Component -- `$e.components.get('document/dynamic')`
+
+*  **Name**: Dynamic.
+*  **Description**: Provide a way to _enable_, _change_, _disable_ dynamic settings, associated with [Dynamic Tags](https://developers.elementor.com/dynamic-tags/),
```
Comment (StyleShit) [inline docs/assets/dev/js/editor/document/dynamic/readme.md:4]:
```suggestion
*  **Description**: Provide a way to _enable_, _change_ or _disable_ dynamic settings, associated with [Dynamic Tags](https://developers.elementor.com/dynamic-tags/),
```

File: docs/assets/dev/js/editor/document/dynamic/readme.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+## Disable _Command_ -- `$e.run('document/dynamic/disable')`
+*  **Name**: Disable.
+*  **Description**: Disable dynamic settings.
+*  **Returns**: `{void}`
+*  **Arguments**:
+
+   | Property     | Type                  | Requirement   | Description |
+   |---           |---                    |---            |---|
+   | _container_  | `{Container}`         | **required**  | Container to disable.
+   | _containers_ | `{Array.<Container>}` | **required**  | Containers to disable.
```
Comment (StyleShit) [inline docs/assets/dev/js/editor/document/dynamic/readme.md:23]: Also in other places.

```suggestion
   | _container_  | `{Container}`         | **required** (if `containers` isn't supplied)  | Container to disable.
   | _containers_ | `{Array.<Container>}` | **required** (if `container` isn't supplied)  | Containers to disable.
```

Comment (StyleShit) [reply docs/assets/dev/js/editor/document/dynamic/readme.md:23]: IDK what's the convention in other places, but maybe change it like that?

```suggestion
   | _container_  | `{Container}`         | **required**  | Container to disable.
   | _containers_ | `{Container[]}` | **required**  | Containers to disable.
```

File: docs/assets/dev/js/editor/document/dynamic/readme.md [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+   | _settings_   | `{Object}`             | **required**  | Dynamic settings to disable <TODO EXPLAIN WHY>
+
+* **Examples**:
```
Comment (StyleShit) [inline docs/assets/dev/js/editor/document/dynamic/readme.md:26]: Also other places:

```suggestion
* **Example**:
```

File: docs/assets/dev/js/editor/document/dynamic/readme.md [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+* **Arguments**:
+
+   | Property     | Type                  | Requirement   | Description | 
+   |---           |---                    |---            |---|
+   | _container_  | `{Container}`         | **required**  | Container target.
+   | _containers_ | `{Array.<Container>}` | **required**  | Containers target.
+   | _settings_   | `{Object}`            | **required**  | Dynamic settings to change.
+
+* **Examples**:
+  Example change dynamic settings for title with dynamic tag *post-date*.
```
Comment (StyleShit) [inline docs/assets/dev/js/editor/document/dynamic/readme.md:94]: You already have "example" in the title

```suggestion
  Change dynamic settings for title with dynamic tag *post-date*.
```

**Review by StyleShit — CHANGES_REQUESTED** [review]

File: docs/assets/dev/js/editor/document/dynamic/readme.md (hunk 2 above)
Comment (iNewLegend) [reply docs/assets/dev/js/editor/document/dynamic/readme.md:23]: Seems like we already added convention like you suggest so we have to follow it now, thanks 👍🏻

File: docs/assets/dev/js/editor/document/component.md [hunk 5]
```diff
@@ -6,6 +6,21 @@
 
   ![legend-section-widget-tabs](_images/legend-section-widget-tabs.png)
 
+Every element in the document has a unique ID, which is used to identify it in the editor, every element has Unique [Container](#) Object, the container object
+can be found by using the following code:
```
Comment (StyleShit) [inline docs/assets/dev/js/editor/document/component.md:10]:
```suggestion
Every element in the document has a unique ID which is used to identify it in the editor, and is resolved to a unique [Container](#) object that can be accessed using the following method:
```

File: docs/assets/dev/js/editor/document/component.md [hunk 6]
```diff
@@ -6,6 +6,21 @@
...
+elementor.getContainer( ID );
+```
+The next example/documents will describe the following parameters:
+
+| Property     | Type                  | Description |
+|---           |---                    |---|
+| _container_  | `{Container}`         | Target.
+| _containers_ | `{Container[]}`       | Targets.
+
+The command(s) can accept container or containers not both.
```
Comment (StyleShit) [inline docs/assets/dev/js/editor/document/component.md:21]: I don't understand the context here

File: docs/assets/dev/js/editor/document/dynamic/readme.md [hunk 7]
```diff
@@ -45,16 +45,16 @@
...
-   | _container_  | `{Container}`         | **required**  | Container to enable.
-   | _containers_ | `{Array.<Container>}` | **required**  | Containers to enable.
-   | _settings_   | `{Object}`            | **required**  | Dynamic settings to enable
+   | Property     | Type             | Description |
+   |---           |---               |---|
+   | _container_  | `{Container}`    | Target.
+   | _containers_ | `{Container[]}`  | Targets.
+   | _settings_   | `{Object}`       | Dynamic settings to enable.
 
 * **Examples**:
```
Comment (StyleShit) [inline docs/assets/dev/js/editor/document/dynamic/readme.md:59]:
```suggestion
* **Example**:
```

File: docs/assets/dev/js/editor/document/dynamic/readme.md [hunk 8]
```diff
@@ -84,19 +84,19 @@
...
-   | _container_  | `{Container}`         | **required**  | Container target.
-   | _containers_ | `{Array.<Container>}` | **required**  | Containers target.
-   | _settings_   | `{Object}`            | **required**  | Dynamic settings to change.
+   | Property     | Type             | Description | 
+   |---           |---               |---|
+   | _container_  | `{Container}`    | Target.
+   | _containers_ | `{Container[]}`  | Targets.
+   | _settings_   | `{Object}`       | Dynamic settings to change.
 
 * **Examples**:
```
Comment (StyleShit) [inline docs/assets/dev/js/editor/document/dynamic/readme.md:93]:
```suggestion
* **Example**:
```

**Review by StyleShit — CHANGES_REQUESTED** [review]

**Review by StyleShit — APPROVED** [review]

**Review by matipojo — CHANGES_REQUESTED** [review]
What is the difference between 
widget-heading-with-dynamic-title-date.png
widget-heading-with-dynamic-title-date-1.png
widget-heading-with-dynamic-title-date-2.png

**Review by matipojo — APPROVED** [review]

**Pull request merged**